docs/src/main/sphinx/admin/properties-web-interface.rst

```diff
@@ -7,7 +7,7 @@ The following properties can be used to configure the :doc:`web-interface`.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * **Type:** :ref:`prop-type-string`
-* **Allowed values:** ``form``, ``fixed``, ``certificate``, ``kerberos``, ``jwt``
+* **Allowed values:** ``form``, ``fixed``, ``certificate``, ``kerberos``, ``jwt``, ``oauth2``
```

Comment:
The possible values are coming from here - https://github.com/trinodb/trino/blob/76b4f01077e4b1a28501e7926e4d9f48a6e2c610/core/trino-main/src/main/java/io/trino/server/ui/WebUiAuthenticationModule.java#L50-L60

So `insecure` is the only one missing (but that's fine I guess).

No change requested.

Reply:
yeah .. I knew but thought there might be reason for not mentioning insecure  ;-)